cmd/explaintest/r/explain_complex_stats.result

```diff
@@ -158,15 +158,15 @@ Projection_5	39.28	root	test.st.cm, test.st.p1, test.st.p2, test.st.p3, test.st.
       └─TableScan_13	160.23	cop	table:st, keep order:false
 explain select dt.id as id, dt.aid as aid, dt.pt as pt, dt.dic as dic, dt.cm as cm, rr.gid as gid, rr.acd as acd, rr.t as t,dt.p1 as p1, dt.p2 as p2, dt.p3 as p3, dt.p4 as p4, dt.p5 as p5, dt.p6_md5 as p6, dt.p7_md5 as p7 from dt dt join rr rr on (rr.pt = 'ios' and rr.t > 1478185592 and dt.aid = rr.aid and dt.dic = rr.dic) where dt.pt = 'ios' and dt.t > 1478185592 and dt.bm = 0 limit 2000;
 id	count	task	operator info
-Projection_10	428.32	root	dt.id, dt.aid, dt.pt, dt.dic, dt.cm, rr.gid, rr.acd, rr.t, dt.p1, dt.p2, dt.p3, dt.p4, dt.p5, dt.p6_md5, dt.p7_md5
+Projection_10	428.32	root	test.dt.id, test.dt.aid, test.dt.pt, test.dt.dic, test.dt.cm, test.rr.gid, test.rr.acd, test.rr.t, test.dt.p1, test.dt.p2, test.dt.p3, test.dt.p4, test.dt.p5, test.dt.p6_md5, test.dt.p7_md5
 └─Limit_13	428.32	root	offset:0, count:2000
-  └─IndexJoin_19	428.32	root	inner join, inner:IndexLookUp_18, outer key:dt.aid, dt.dic, inner key:rr.aid, rr.dic
+  └─IndexJoin_19	428.32	root	inner join, inner:IndexLookUp_18, outer key:test.dt.aid, test.dt.dic, inner key:test.rr.aid, test.rr.dic
     ├─TableReader_41	428.32	root	data:Selection_40
-    │ └─Selection_40	428.32	cop	eq(dt.bm, 0), eq(dt.pt, "ios"), gt(dt.t, 1478185592), not(isnull(dt.dic))
+    │ └─Selection_40	428.32	cop	eq(test.dt.bm, 0), eq(test.dt.pt, "ios"), gt(test.dt.t, 1478185592), not(isnull(test.dt.dic))
     │   └─TableScan_39	2000.00	cop	table:dt, range:[0,+inf], keep order:false
```

Comment:
Is it `dt` is alias name? If it's true, it is called `dt` here, but it's called `test.dt` on line 165. Is this a bit confused? @kennytm

Reply:
@eurekaka

Reply:
You mean we should use `test.dt` in line 166? Currently, we do not print database name for PhysicalTableScan.

```golang
func (p *PhysicalTableScan) ExplainInfo() string {
    buffer := bytes.NewBufferString("")
    tblName := p.Table.Name.O
    if p.TableAsName != nil && p.TableAsName.O != "" {
        tblName = p.TableAsName.O
    }
    fmt.Fprintf(buffer, "table:%s", tblName)
    ...
}
```
If needed, we can print DBName as well, since it is recorded in `PhysicalTableScan`.

Reply:
@eurekaka 
No, I want to say if this situation will make you feel troubled.

Reply:
Oh I think it is not bothering since we merely meet queries where 2 tables share same name but from different databases.

Reply:
Got it.